Makefile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Configuration variables
+#
+VENV ?= .venv
+export VIRTUAL_ENV := $(abspath ${VENV})
+export PATH := ${VIRTUAL_ENV}/bin:${PATH}
+
+# Use the command from https://stackoverflow.com/a/35730328/5601796
+.PHONY: help
+help:
```

Comment:
the `help:` target deserves a comment as well  or it will not show up in the output of `make`

"`help: ## Display the Make targets`" and the other help texts should start with an Upper-case :-)

Reply:
Thanks, done.